### DISCUSSION TITLE
data format

### DISCUSSION THREAD

**Discussion by yukinobu-mat** [mtex-toolbox/mtex#571] [Ask Anything] [answered]
Hello All,

I am trying to calculate the GND for an.crc file
However during the loading and running the code I face some problems.
Error: GND (line 22)
ipfKey = ipfHSVKey(ebsd);

Probably I think I'm using the wrong file.
I use Aztec software ( Oxford Instruments ) .
What information do I need to calculate GND ?
And, What format should the file be in?

I want to inform taht I use MTEX 5.4.0

Much appreciated your help.
Many regards,
Yukinobu

**Comment by kilir** [comment]
Hi,
type `ebsd` in the command window after loading the data. Also, the error message should not be only this single line, right?
Cheers,
Rüdiger

**Reply by yukinobu-mat** [reply]
Thank you for your reply.
as you said, there are some error message.
This is all error messages.
I missed it, does this mean that the data needs to be modified for a single phase?

----------------------------------------------------------------
Your variable contains the phases: notIndexed, Ni-superalloy

However, your are executing a command that is only permitted for a single phase!

Please see modify EBSD data for how to restrict EBSD data or grains to a single phase.
----------------------------------------------------------------


Error: phaseList/get.CS (line 154)
      id = checkSinglePhase(pL);

Error: EBSD/subsref (line 40)
  [varargout{1:nargout}] = builtin('subsref',ebsd,s);

Error: orientationColorKey (line 23)
        oCK.CS1 = ebsd.CS;

Error: ipfColorKey (line 13)
      oM = oM@orientationColorKey(varargin{:});

Error: ipfHSVKey (line 16)
    oM = oM@ipfColorKey(varargin{:});

Error: GND (line 22)
ipfKey = ipfHSVKey(ebsd);

**Comment by kilir** [comment, marked as answer]
Yes.
Use for example `ebsd('Ni')` instead of `ebsd`.
Have a look here https://mtex-toolbox.github.io/EBSDSelect.html

Cheers,
Rüdiger

**Reply by yukinobu-mat** [reply]
Thank you !
I'll try it soon !

Regards
Yukinobu
